R/convertMT.R

```diff
@@ -0,0 +1,115 @@
+#' @title Convert migration times to effective mobility
+#'
+#' @description
+#'
+#' `convertMT` uses a list of known EOF markers with to convert migration times 
+#' (MTs) to effective mobilities (µeff). By using the information of the EOF 
+#' markers (which is their MT and µeff), it is possible to transform the MT-
+#' scale (for consistency called rtime) into the µeff scale. This is used in 
+#' CE-MS to overcome MT shifts originating from EOF fluctuations in different 
+#' runs.
+#'
+#' @param x `numeric` vector with migration times in minutes.
+#'
+#' @param y `data.frame`data.frame containing minimum two columns, where one
+#'     holds the determined migration time in minutes (here referred to as 
```

Comment:
The time is in minute yes, since the unit of the effective mobility is `mm^2 / (kV * min)`

Previously I included a conversion function `rtime <- rtime/60` in it but Laurent suggested to remove it

Reply:
The time is in minute yes, since the unit of the effective mobility is mm^2 / (kV * min)

Previously I included a conversion function rtime <- rtime/60 in it but Laurent suggested to remove it

Reply:
OK then.